tests/core_scheduler/test_rollback_cli_e2e.py

```
@@ -0,0 +1,106 @@
+    httpd = HTTPServer(("127.0.0.1", port), _Handler)
+    t = threading.Thread(target=httpd.serve_forever, daemon=True)
+    t.start()
+    try:
+        env = dict(os.environ)
+        env["AIRTABLE_API_KEY"] = "pat_test"
+        env["AIRTABLE_BASE_ID"] = "appTest"
+        env["AIRTABLE_TABLE_ID"] = "tblLKMcDh7buNC9tr"
+        env["AIRTABLE_API_BASE_URL"] = f"http://127.0.0.1:{port}"
+
+        proc = subprocess.run(
+            ["python3", "-m", "modules.core_scheduler", "rollback", "--run-id", "RUN-1", "--dry-run"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)